spring-security/src/main/java/com/giffing/examples/wicket/spring/boot/springsecurity/WicketWebSecurityApapterConfig.java

```diff
@@ -1,13 +1,22 @@
 package com.giffing.examples.wicket.spring.boot.springsecurity;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
 import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 
 @Configuration
 public class WicketWebSecurityApapterConfig extends WebSecurityConfigurerAdapter {
+
+	@Override
+	@Bean(name = "authenticationManager")
+	public AuthenticationManager authenticationManagerBean() throws Exception {
+		return super.authenticationManagerBean();
+	}
```

Comment:
Spring Boot 2 doesn't provide the `authenticationManager` bean automatically anymore when it finds the `WebSecurityConfigurerAdapter` on the classpath.

See this issue for details: https://github.com/spring-projects/spring-boot/issues/11136